Makefile

```diff
@@ -3,83 +3,62 @@
 .PHONY: all
 all: help
 
-.PHONY: venv
-venv:
-	python3 -m venv .venv ## Provision a Python 3 virtualenv for **development**
-	.venv/bin/pip install --upgrade pip wheel
-	.venv/bin/pip install --require-hashes -r "requirements/dev-requirements.txt"
-
 .PHONY: bandit
 bandit: ## Run bandit with medium level excluding test-related folders
-	pip install --upgrade pip && \
-		pip install --upgrade bandit!=1.6.0 && \
-	bandit -ll --recursive securedrop_proxy
+	@echo "Installing latest version of Bandit in virtual environment…"
+	@poetry update bandit
+	@echo "Running bandit security checks…"
+	@poetry run bandit -ll --recursive securedrop_proxy
 
 .PHONY: safety
 safety: ## Runs `safety check` to check python dependencies for vulnerabilities
-	pip install --upgrade safety && \
-		for req_file in `find . -type f -name '*requirements.txt'`; do \
-			echo "Checking file $$req_file" \
-			&& safety check --full-report -r $$req_file \
-			&& echo -e '\n' \
-			|| exit 1; \
-		done
+	@echo "Installing latest version of safety in virtual environment…"
+	@poetry update safety
+	@echo "Running safety against requirements files in this repository…"
+	for req_file in `find . -type f -name '*requirements.txt'`; do \
+		echo "Checking file $$req_file" \
+		&& poetry run safety check --full-report -r $$req_file \
```

Comment:
If we keep `safety` around at all, I'd like to of course point it at the Poetry lockfile. Unfortunately that's not possible yet (the good old `category` change broke `safety` as well: https://github.com/pyupio/safety/issues/479), so we do lose some `safety` check functionality here (but we can turn on Dependabot to make up for it).